unicodetools/src/test/java/org/unicode/text/UCD/TestTestUnicodeInvariants.java

```diff
@@ -40,4 +40,10 @@ void testSecurityInvariants() throws IOException {
         int rc = TestUnicodeInvariants.testInvariants("SecurityInvariantTest.txt", true);
         assertEquals(0, rc, "TestUnicodeInvariants.testInvariants(security) failed");
     }
+
```

Comment:
Did this leak from your other work?

Reply:
Looks like the base for the diff was before that one was merged into main (I had merged it in here separately beforehand because I wanted to see whether the UCD invariants were happy).

The diff should be clean now.

Reply:
Hm, no, something weirder had happened, somehow this test function got duplicated in a merge.

Well, _now_ the diff should be clean…